{"name": "Test generate token with invalid accounts", "status": "passed", "parameters": [{"name": "account", "value": "<data_object.account.Account object at 0x00000167441EFE60>"}], "start": 1702368257406, "stop": 1702368258984, "uuid": "f3cb0bcc-86cf-4217-880d-fe0e78c6e9fa", "historyId": "fc2f68864449eebe1721d55d70e89577", "testCaseId": "573dcf4d17f28e829fe2a39fec07d16d", "fullName": "testcases.api.test_generate_token#test_generate_token_unsuccessfully_with_invalid_account", "labels": [{"name": "suite", "value": "Test Generate Token"}, {"name": "parentSuite", "value": "API Test"}, {"name": "subSuite", "value": "TestGenerateToken"}, {"name": "host", "value": "VNNOT02222"}, {"name": "thread", "value": "14724-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api.test_generate_token"}]}